src/sentry/utils/integrationdocs.py

```diff
@@ -122,9 +123,19 @@ def sync_docs(quiet=False):
 
     dump_doc("_platforms", {"platforms": platform_list})
 
+    # This value is derived from https://docs.python.org/3/library/concurrent.futures.html#threadpoolexecutor
+    MAX_THREADS = 32
```

Comment:
This isn't used now, the linter will catch it in CI but saving you some scrolling.

Reply:
Fixed, sorry :D